custom/icds_reports/templates/icds_reports/icds_app/mobile/kpi.directive.html

```diff
@@ -1,45 +1,25 @@
-                </div>
-                <div class="col-md-11 kpi-content">
-                    <div class="kpi-justify-center">
-                        <span class="title white" ng-bind="cell.label"></span>
-                    </div>
-                    <div class="kpi-justify-center">
-                        <span class="number white" ng-if="cell.format == 'div'"><span class="white" ng-bind="cell.value | indiaNumbers"></span>/<span class="white" ng-bind="cell.all | indiaNumbers"></span></span>
-                        <span class="number white" ng-if="cell.format == 'number'" ng-bind="cell.value | indiaNumbers"></span>
-                        <span class="string white" ng-if="cell.format == 'string'" ng-bind="cell.value"></span>
-                        <div class="kpi-numbers-format" ng-if="cell.format == 'percent_and_div'">
-                            <div class="number white"><span class="white" ng-bind="(cell.value / (cell.all || 1) * 100) | number:2"></span>%</div>
-                            <div class="second_number white">(<span class="white" ng-bind="cell.value | indiaNumbers"></span> / <span  class="white" ng-bind="cell.all | indiaNumbers"></span>)</div>
-                        </div>
-                        <div class="kpi-numbers-format" ng-if="cell.format == 'number_and_percent'">
-                            <div class="number white"><span class="white" ng-bind="cell.value | indiaNumbers"></span></div>
-                            <div class="second_number white"><span class="white" ng-bind="(cell.value / (cell.all || 1) * 100) | number:2"></span>%</div>
-                        </div>
-                    </div>
-                    <div class="kpi-justify-center" ng-if="cell.show_percent === void(0)"
-                         ng-class="{'kpi-percent': cell.redirect}">
-                        <span class="white" ng-class="{'kpi-percents-text': cell.redirect}"
-                              ng-show="cell.percent !== null && $ctrl.isNumber(cell.percent) && $ctrl.showPercentInfo()">
-                            <i class="fa" style="font-size: 1.5em" ng-show="cell.percent !== 0"
-                               ng-class="{'fa-arrow-down': cell.percent < 0, 'fa-arrow-up': cell.percent > 0, 'green': cell.color === 'green', 'red': cell.color === 'red'}" ></i>&nbsp;
-                            <span class="white" ng-bind="cell.percent | number:2"></span>% from previous {$ cell.frequency $}
-                        </span>
-                        <span class="white" ng-class="{'kpi-percents-text': cell.redirect}"
-                              ng-show="cell.percent !== null && !$ctrl.isNumber(cell.percent) && $ctrl.showPercentInfo()">
-                            <span class="white" ng-bind="cell.percent"></span>
-                        </span>
-                    </div>
-                </div>
+{% load staticfiles%}
+<div ng-repeat="section in $ctrl.data track by $index">
```

Comment:
most of this copy/pasted from https://github.com/dimagi/mobile-dashboard-prototype/pull/8